### ISSUE TITLE
GLPI 10.0.0 - Problem with polish font in the software

### ISSUE THREAD

**Issue by WebGregGit** [glpi-project/glpi#11468]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.0

### Bug description

The software is not displayed with correct polish characters. There was no problem in 9.5.6

But the bigger problem for me is that the software tab slowed down extremely after updating to 10.0.0

### Relevant log output

```shell
php-errors.log

[2022-05-02 09:00:02] glpiphplog.WARNING:   *** PHP Warning (2): ldap_search(): Partial search results returned: Sizelimit exceeded in /var/www/html/glpi/src/AuthLDAP.php at line 1867
  Backtrace :
  src/AuthLDAP.php:1867                              ldap_search()
  src/AuthLDAP.php:2025                              AuthLDAP::searchForUsers()
  src/Console/Ldap/SynchronizeUsersCommand.php:291   AuthLDAP::getAllUsers()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Ldap\SynchronizeUsersCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()





sql-errors.log

[2022-05-02 07:47:49] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: INSERT INTO `glpi_devicefirmwares` (`date`, `version`, `manufacturers_id`, `designation`, `devicefirmwaretypes_id`, `date_creation`, `date_mod`) VALUES ('2018-13-06', 'K51 v01.86', '608', 'Hewlett-Packard BIOS', '1', '2022-05-02 07:47:48', '2022-05-02 07:47:48')
  Error: Incorrect date value: '2018-13-06' for column 'date' at row 1
  Backtrace :
  src/DBmysql.php:1310                               
  src/CommonDBTM.php:682                             DBmysql->insert()
  src/CommonDBTM.php:1274                            CommonDBTM->addToDB()
  src/CommonDevice.php:507                           CommonDBTM->add()
  src/Inventory/Asset/Device.php:113                 CommonDevice->import()
  src/Inventory/Asset/Bios.php:85                    Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Bios->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"} 
[2022-05-02 07:56:26] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: DELETE `glpi_items_devicesoundcards` FROM `glpi_items_devicesoundcards` WHERE `devicesoundcards_id` = '2' AND `items_id` = '222' AND `itemtype` = 'Computer'
  Error: Deadlock found when trying to get lock; try restarting transaction
  Backtrace :
  src/DBmysql.php:1540                               
  src/Inventory/Asset/Device.php:122                 DBmysql->delete()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"} 
[2022-05-02 08:08:54] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: DELETE `glpi_items_devicesoundcards` FROM `glpi_items_devicesoundcards` WHERE `devicesoundcards_id` = '2' AND `items_id` = '330' AND `itemtype` = 'Computer'
  Error: Deadlock found when trying to get lock; try restarting transaction
  Backtrace :
  src/DBmysql.php:1540                               
  src/Inventory/Asset/Device.php:122                 DBmysql->delete()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"} 
[2022-05-02 08:17:30] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: DELETE `glpi_items_devicesoundcards` FROM `glpi_items_devicesoundcards` WHERE `devicesoundcards_id` = '2' AND `items_id` = '226' AND `itemtype` = 'Computer'
  Error: Deadlock found when trying to get lock; try restarting transaction
  Backtrace :
  src/DBmysql.php:1540                               
  src/Inventory/Asset/Device.php:122                 DBmysql->delete()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"} 
[2022-05-02 08:22:55] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: INSERT INTO `glpi_items_devicesoundcards` (`devicesoundcards_id`, `itemtype`, `items_id`, `is_dynamic`, `locations_id`, `entities_id`, `is_recursive`) VALUES ('33', 'Computer', '631', '1', '0', '0', '0')
  Error: Deadlock found when trying to get lock; try restarting transaction
  Backtrace :
  src/DBmysql.php:1310                               
  src/CommonDBTM.php:682                             DBmysql->insert()
  src/CommonDBTM.php:1274                            CommonDBTM->addToDB()
  src/Inventory/Asset/Device.php:134                 CommonDBTM->add()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"} 
[2022-05-02 08:46:38] glpisqllog.ERROR: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 368
  *** MySQL query error:
  SQL: DELETE `glpi_items_devicesoundcards` FROM `glpi_items_devicesoundcards` WHERE `devicesoundcards_id` = '33' AND `items_id` = '476' AND `itemtype` = 'Computer'
  Error: Deadlock found when trying to get lock; try restarting transaction
  Backtrace :
  src/DBmysql.php:1540                               
  src/Inventory/Asset/Device.php:122                 DBmysql->delete()
  src/Inventory/Asset/MainAsset.php:772              Glpi\Inventory\Asset\Device->handle()
  src/Inventory/Asset/MainAsset.php:693              Glpi\Inventory\Asset\MainAsset->handleAssets()
  src/RuleImportAsset.php:939                        Glpi\Inventory\Asset\MainAsset->rulepassed()
  src/Rule.php:1505                                  RuleImportAsset->executeActions()
  src/RuleCollection.php:1589                        Rule->process()
  src/Inventory/Asset/MainAsset.php:519              RuleCollection->processAllRules()
  src/Inventory/Inventory.php:678                    Glpi\Inventory\Asset\MainAsset->handle()
  src/Inventory/Inventory.php:316                    Glpi\Inventory\Inventory->handleItem()
  src/Inventory/Request.php:364                      Glpi\Inventory\Inventory->doInventory()
  src/Inventory/Request.php:88                       Glpi\Inventory\Request->inventory()
  src/Agent/Communication/AbstractRequest.php:323    Glpi\Inventory\Request->handleAction()
  src/Agent/Communication/AbstractRequest.php:235    Glpi\Agent\Communication\AbstractRequest->handleJSONRequest()
  front/inventory.php:83                             Glpi\Agent\Communication\AbstractRequest->handleRequest()
  {"user":"inventory@glpi"}
```

### Steps To reproduce

1. Install some polish software on your computer (ex. https://www.ips-infor.com.pl/pliki/pityipssetup.exe - if you choose to install, the error is not in the program name, but in the publisher and in the comment)
2. Take an inventory with agent glpi 1.2
3. Go to Assets > Computers > Software tab > check polish characters for software you installed

### Your GLPI setup information

GLPI 10.0.0 ( => /var/www/html/glpi)
Installation mode: TARBALL
Current language:en_GB

Operating system: Linux glpi 5.4.0-105-generic #119-Ubuntu SMP Mon Mar 7 18:49:24 UTC 2022 x86_64
PHP 7.4.3 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml,
	mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, snmp, soap, sockets, sodium, standard,
	sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="1200" memory_limit="128M" post_max_size="10M" safe_mode="" session.save_handler="files"
	upload_max_filesize="10M" 
Software: Apache/2.4.41 (Ubuntu) (Apache/2.4.41 (Ubuntu) Server at glpi.ad.city.pl Port 443
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.88 Safari/537.36
Server Software: (Ubuntu)
	Server Version: 8.0.28-0ubuntu0.20.04.3
	Server SQL Mode: STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
	Parameters: glpiuser@localhost/glpidb
	Host info: Localhost via UNIX socket
	
PHP version (7.4.3) is supported.PHP version (7.4.3) is supported.
Sessions configuration is OK.Sessions configuration is OK.
Allocated memory is sufficient.Allocated memory is sufficient.
mysqli extension is installed.mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.curl extension is installed.
gd extension is installed.gd extension is installed.
intl extension is installed.intl extension is installed.
libxml extension is installed.libxml extension is installed.
zlib extension is installed.zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (8.0.28) is supported.Database engine version (8.0.28) is supported.
The log file has been created successfully.The log file has been created successfully.
Write access to /var/www/html/glpi/files/_cache has been validated. Write access to /var/www/html/glpi/config has been validated. Write access to /var/www/html/glpi/files/_cron has been validated. Write access to /var/www/html/glpi/files has been validated. Write access to /var/www/html/glpi/files/_dumps has been validated. Write access to /var/www/html/glpi/files/_graphs has been validated. Write access to /var/www/html/glpi/files/_lock has been validated. Write access to /var/www/html/glpi/files/_pictures has been validated. Write access to /var/www/html/glpi/files/_plugins has been validated. Write access to /var/www/html/glpi/files/_rss has been validated. Write access to /var/www/html/glpi/files/_sessions has been validated. Write access to /var/www/html/glpi/files/_tmp has been validated. Write access to /var/www/html/glpi/files/_uploads has been validated.Write access to /var/www/html/glpi/files/_cache has been validated.
Write access to /var/www/html/glpi/config has been validated.
Write access to /var/www/html/glpi/files/_cron has been validated.
Write access to /var/www/html/glpi/files has been validated.
Write access to /var/www/html/glpi/files/_dumps has been validated.
Write access to /var/www/html/glpi/files/_graphs has been validated.
Write access to /var/www/html/glpi/files/_lock has been validated.
Write access to /var/www/html/glpi/files/_pictures has been validated.
Write access to /var/www/html/glpi/files/_plugins has been validated.
Write access to /var/www/html/glpi/files/_rss has been validated.
Write access to /var/www/html/glpi/files/_sessions has been validated.
Write access to /var/www/html/glpi/files/_tmp has been validated.
Write access to /var/www/html/glpi/files/_uploads has been validated.
Web access to the files directory should not be allowed but this cannot be checked automatically on this instance. Make sure access to error log file (/files/_log/php-errors.log) is forbidden; otherwise review .htaccess file and web server configuration.Web access to the files directory should not be allowed but this cannot be checked automatically on this instance.
Make sure access to error log file (/files/_log/php-errors.log) is forbidden; otherwise review .htaccess file and web server configuration.
exif extension is installed.exif extension is installed.
ldap extension is installed.ldap extension is installed.
openssl extension is installed.openssl extension is installed.
zip extension is installed.zip extension is installed.
bz2 extension is installed.bz2 extension is installed.
Zend OPcache extension is installed.Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodiad.Following extensions are installed: ctype, iconv, mbstring, sodiad.
Write access to /var/www/html/glpi/marketplace has been validated.Write access to /var/www/html/glpi/marketplace has been validated.
Timezones seems loaded in database.Timezones seems loaded in database.

Plugins list
--
datainjection        Name: Data injection                 Version: 2.9.0      State: Installed / not activated 	
glpiinventory        Name: GLPI Inventory                 Version: 1.0.0      State: Installed / not activated 	
printercounters      Name: Printer counters               Version: 1.7.0      State: Installed / not activated 	
treeview             Name: Tree view                      Version: 1.9.1      State: Installed / not activated 	
webresources         Name: Web Resources                  Version: 1.3.2      State: Installed / not activated 	
tag                  Name: Zarządzanie słowami kluczowymi Version: 2.8.1      State: Installed / not activated

### Anything else?

The second problem is that when i go to front/software.php I have a blank page for a few minutes. When checking on the server, mysql takes up 100% of the CPU. I have over 11k software items, but in 9.5.6 there was no problem with it - the page loaded in a second (even when I set to display 1000 per page). [This is what it looks like when I go to /front/software.php](https://zapodaj.net/images/d7bf4acec9c46.jpg) - a view of the white html page and what it looks like from the server side.

I don't know if the problem relates (maybe I should to send this as a new issue). It may be related to polish characters, because the version immediately after upgrading to 10, when GLPI agents had not yet sent the data (i.e. there were no entries with a defective polish font), it displayed faster. Not as good as 9.5.6, but faster.

[This is how the agent's html log file looks like on a workstation](https://zapodaj.net/images/0f13990512eb6.jpg) - no problem with polish fonts.

[This is the software tab for computer on GLPI](https://zapodaj.net/images/86a4770188b89.jpg)

[When the page finally loads, you can see that the entries are multiplied](https://zapodaj.net/images/963e247a04090.jpg)

**Comment by cedric-anne** [comment]
Hi,

Can you attach here the inventory file corresponding to the related computer ?

**Comment by WebGregGit** [comment]
Hi @cedric-anne 

I don't necessarily want to release it publicly. Do you have any secure channel? Or if there is no other option, maybe only some of the data you need - what kind of data?

**Comment by WebGregGit** [comment]
For the test:

`DELETE FROM glpi_softwares`

and deleted the computer - re-inventorying - the problem persists. But not in the agent log file (here I see polish fonts) - in the GLPI itself.


Maybe the problem is with the font you set? However, that does not explain why some of the data is displayed correctly. However, when I go to front/preference.php > Personalization > and change the font - I don't see the effect of this change.

**Comment by trasher** [comment]
Private channels are included in professional support.

You can limit inventory to only relevant parts (see agents docs) and anonymize data on your side before sending.

**Comment by WebGregGit** [comment]
OK but what parts are relevant for you? Software only?

**Comment by trasher** [comment]
Yes; at least one that permit to reproduce your issue

**Comment by WebGregGit** [comment]
I am sending the file from the file to which the exe link I gave in the first post. But as you can see here, the Polish font is displaying correctly
[software_short.txt](https://github.com/glpi-project/glpi/files/8602026/software_short.txt)
.

**Comment by WebGregGit** [comment]
I found in history:

<html>
<body>


1558616 | 2022-01-03 03:01 | Plugin_FusionInventory | Software | Uninstall a software: Dialog poÂwiadczeÃ¤ - 10.0.15063.0 (12257)
-- | -- | -- | -- | --



</body>
</html>


So it does not problem with new data downloaded by the agent. But again - why only part of the data? Why, when I am on the 'front/software.php' page, I can see e.g. 

<html>
<body>


Dialog poświadczeń | Poczta e-mail, telefon lub Skype | 10.0.17763.110.0.19041.42310.0.18362.26710.0.19041.1023
-- | -- | --
Dialog poÅwiadczeÅ | Microsoft | 10.0.18362.38710.0.19041.102310.0.18362.449


</body>
</html>

**Comment by trasher** [comment]
Just tested with provided file, seems I cannot reproduce:
![image](https://user-images.githubusercontent.com/224733/166233361-c8a9e671-fdf9-41bc-90cf-294b4a4d17e7.png)

Double check your HTTP server configuration, in case there is something wrong in there regarding charsets.

**Comment by WebGregGit** [comment]
Thanks for the verification. Can you also check this? 

```
         {
            "arch": "neutral",
            "folder": "C:\\Windows\\SystemApps\\microsoft.creddialoghost_cw5n1h2txyewy",
            "from": "uwp",
            "install_date": "2021-07-02",
            "name": "Dialog poświadczeń",
            "publisher": "Microsoft Corporation",
            "system_category": "system",
            "version": "10.0.19041.1023"
         },
```

![mysql2](https://user-images.githubusercontent.com/68922721/166255719-33449b20-5931-465f-8deb-d8cb45e7efec.jpg)



I haven't changed anything in the configuration except upgrading GLPI. Version 9 - perfect. Version 10 - problems.

**Comment by trasher** [comment]
OK on my side as well:
![image](https://user-images.githubusercontent.com/224733/166257010-6ebd0e15-3f83-48ff-838b-aa25b672b431.png)

How did you proceed the update? Did you copy the 9.5 configuration file to 10.0 directory and then run update? Had you any errors?
This may be a connection charset issue; but I do not know how to check that.

**Comment by WebGregGit** [comment]
1. Stoped apache
2. Copy glpi folder
3. Delete glpi folde
4. Downloaded release
5. Unpacked
6. cd /var/www/html/
7. sudo chmod -Rf 775 *
8. sudo chown -Rf www-data. *
9. Copied back folders: config, files, marketplace and plugins directory
10. sudo php bin/console db:update
11. Start apache
12. And... OK if I remember correctly there was a message about to execute the command:
php bin/console glpi:migration:utf8mb4

This could be it.
I will restore the backup today and perform the update again and show you the result. There were a couple of other suggestions there

**Comment by WebGregGit** [comment]
I have to apologize - version 9 wasn't perfect. The error was not actually present in the computers I was checking, but there were already such cases throughout the database. After uprade to 10, there are simply many more of them.

This time, I deleted all the faulty entries first.
I didn't use the command: sudo php bin/console db:update but upgraded from www.
Below is the result.

![update01](https://user-images.githubusercontent.com/68922721/166294271-ea22c126-005f-4b3c-87be-1a035c7ffb9f.jpg)

![update02](https://user-images.githubusercontent.com/68922721/166294284-a14e9b92-1383-4493-8cdf-9ac34cdeb0fb.jpg)

If you look at the previous screenshot (mysql), there are two entries - id = 11901 and 7195.

7195 was version 9 - I deleted it before this upgrade.

Immediately after updating to 10 (I didn't take any action apart from updating the database and logging in to the website), I checked these entries again - a new 11935 (equivalent to the previous 11901) appeared.

**Comment by WebGregGit** [comment]
Hi

I did more tests today. One of them was offline. This time, right after the upgrade, it was ok. I started apache, forced inventory from problematic computer - badly coded characters appeared. But going to the software page resulted in a blank page, mysql at 100% cpu and a few minutes of waiting for the page to load. I wasn't sure if it was mysql or agent.

Next test - I updated the agent from GLPI 1.1.nightly to 1.2. So far it is good.

Maybe it was an agent problem.
